scripts/monitoring/cron-send-docker-dns-resolution.py

```diff
@@ -21,8 +21,14 @@
 
 if __name__ == "__main__":
     cli = AutoVersionClient(base_url='unix://var/run/docker.sock')
-    container = cli.create_container(image='docker-registry.ops.rhcloud.com/ops/oso-rhel7-host-monitoring',
+
+    container_id = ''
+    for con in cli.containers(filters={'name': 'oso-rhel7-host-monitoring'}):
```

Comment:
Can we use a pattern here?

Don't forget that we have centos containers that can run this code too.

Also, what happened to getting the container_id from the environment since we're running inside of the container?